mediacenter/tests/test_search.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from search.models import Search
+
+from ..models import Document
+
+pytestmark = pytest.mark.django_db
+
+
+def test_document_are_indexed(video):
+    assert Search.objects.count() == 1
+    assert len(list(Search.search(public=True))) == 1
+    assert Document.objects.search("Ikinyugunyugu").count() == 0
```

Comment:
this is the same for every test where your start by checking that there is something at the start of the function